.circleci/config.base.yml

```diff
@@ -431,20 +361,7 @@ workflows:
                 - /tagged-release\/.*/
                 - /run-e2e\/.*/
           requires:
-            - build
```

Comment:
Build wasn't required here, since we already have `publish_to_local_registry` which DOES depend on build. This just gets rid of some unnecessary line in the dep graph for our test runs.